src/api/apis/MALAPIManga.ts

```diff
@@ -12,7 +12,7 @@ export class MALAPIManga extends APIModel {
 		super();
 
 		this.plugin = plugin;
-		this.apiName = 'MALAPI Manga';
+		this.apiName = 'MALAPIManga';
```

Comment:
You should leave the API names as they are since they are shown to the user in the advanced search modal.
It would be best if you added a new id field to the APIs and used that instead.